stdlib/auth/src/main/ballerina/auth/jwt_auth_provider.bal

```diff
@@ -123,13 +130,15 @@ const string AUTH_TYPE_JWT = "jwt";
 # + certificateAlias - Token signed key alias
 # + trustStoreFilePath - Path to the trust store file
 # + trustStorePassword - Trust store password
+# + jwtCache - Cache used to store parsed JWT information as CachedJwt
 public type JWTAuthProviderConfig record {
     string issuer = "";
     string audience = "";
     int clockSkew = 0;
     string certificateAlias = "";
     string trustStoreFilePath = "";
     string trustStorePassword = "";
+    cache:Cache jwtCache = new;
```

Comment:
For Ballerina programmers to access and use the cached records, we need to make `type CachedJWTAuthContext` also `public`. Since it does not make much sense to name it `CachedJWTAuthContext`, we renamed it to `CachedJwt` too.

Reply:
Thanks for pointing out. I missed that part. Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/14847/commits/70278a8b9aa89fcc709111d1f79b002334c6f8ab